blues/simulation.py

```diff
@@ -47,7 +48,19 @@ def __init__(self, structure, move_engine, **opt):
         self.nc  = None
 
         self.opt = opt
-    def generateAlchSystem(self, system, atom_indices):
+
+    def _zero_allother_masses(self, system, indexlist):
+        num_atoms = system.getNumParticles()
+        for index in range(num_atoms):
+            if index in indexlist:
+                pass
+            else:
+                system.setParticleMass(index, 0*unit.daltons)
+        return system
+
+    def generateAlchSystem(self, system, atom_indices,
+                            freeze_distance=0, freeze_center='LIG', freeze_solvent='HOH,NA,CL',
```

Comment:
Do we have appropriate disclaimers in the READMEs about what is frozen and when that would likely cause problems? e.g. freezing solvent will be a problem in a solvent exposed binding site, right?

Reply:
That is a good point. I'll have to think about how to best handle this.